client/galaxy/scripts/mvc/citation/citation-model.js

```diff
@@ -14,9 +14,9 @@ var logNamespace = 'citation';
 var Citation = Backbone.Model.extend( baseMVC.LoggableMixin ).extend( {
     _logNamespace : logNamespace,
 
-    initialize: function( ) {
-        var bibtex = this.attributes.content;
-        var entry = new bibtex.BibtexParser(bibtex).entries[0];
+    initialize: function() {
+        var bibtex = this.get( 'content' );
+        var entry = new BibtexParser(bibtex).entries[0];
```

Comment:
Looks like it still worked because a 'new' copy of the singleton form (`BibtexParser(string)`) still will function as the original. /shrug

In any event, I'll remove the new. One sec.